### ISSUE TITLE
Issue titles in issues tab are dimmed / gray

### ISSUE THREAD

**Issue by makeworld-the-better-one** [refined-github/refined-github#5299]
### Please ensure:

- [X] The bug is caused by Refined GitHub. It doesn't happen if I disable the extension.

### Example URLs

https://github.com/makeworld-the-better-one/amfora/issues

### Browser(s) used

Firefox

### Description

Issue titles are grayed out, only in the issues tab:

![image](https://user-images.githubusercontent.com/25111343/149390274-c8fdd7a0-b91d-4551-b5dd-60a7f40620b7.png)

### Include in this issue:

- [X] Screenshots/video/gif demonstrating the bug, if it’s visual
- [ ] Console errors, if any

**Comment by yakov116** [comment]
Its a feature
https://github.com/refined-github/refined-github/pull/5258

To undo put the following in custom CSS box
```css
.js-issue-row .h4.h4:visited:not(:hover) {
	color: var(--color-fg-default) !important;
}
```

**Comment by makeworld-the-better-one** [comment]
Thanks for explaining. Shouldn't this be a preference though, that can be easily toggled?

**Comment by yakov116** [comment]
There are many small refinements we make, not everything will have an option to toggle it off.

**Comment by makeworld-the-better-one** [comment]
I'd argue this is something that should be toggleable, as it can have the large effect of making all the issues in a repo hard to read. Anyway, it's up to the team I guess. Thanks for the fix.

**Comment by reidpr** [comment]
Thanks for the workaround. In my case the titles load as black first, then change to gray, causing a distracting flicker in addition to making the text harder to read.

**Comment by jhpratt** [comment]
Given that there are over 200 options, one more doesn't hurt. I find this to be a _significant_ regression in usability.

**Comment by kidonng** [comment]
My sincere apology to everyone troubled by this change. It was proposed by me, and once more full-fledged standalone feature until [`e344eac` (#5258)](https://github.com/refined-github/refined-github/pull/5258/commits/e344eac73e6c948ba001bd35f33c1aef49b2d32a).

I have commited a hotfix to revert the style, so you won't see it anymore. In the future I will try to be more considerate and don't just push features onto users.

**Comment by kidonng** [comment]
> I'd argue this is something that should be toggleable, as it can have the large effect of making all the issues in a repo hard to read.

Thanks for you explanation and especially the screenshot, I can see why it matters now: if you have write access to a repo (e.g. you are the owner), you are probably going to visit most (if not all) of the conversations, making them become dimmed.

In the linked PR I proposed a fix to prevent this situation.

> In my case the titles load as black first, then change to gray, causing a distracting flicker in addition to making the text harder to read.

I'm sorry to hear that but this sounds like a bug we should fix. Could you please provide a screen recording if possible?

> I find this to be a _significant_ regression in usability.

First, please note this is how titles in Discussions ([example](https://github.com/vercel/vercel/discussions)) already are. GitHub's is even worse since the titles are still gray even when hovered.

Second, there's not much alternative styling we can do due to [browser limitations](https://developer.mozilla.org/en-US/docs/Web/CSS/Privacy_and_the_:visited_selector#limits_to_visited_link_styles).

Third, this style doesn't dim the entire conversation, just the title, which is bold, and would remain recognizable most of the time.

I hope you understand the intention of this style is to _improve_ usability, in the sense that it helps you identify which issues you have visited or not. Again, I want to stress that it was implemented as a more useful and less annoying feature. You are welcomed to read the #5258 for more context.

**Comment by makeworld-the-better-one** [comment]
I think your fix is probably a good idea. I see that it also makes it a toggleable option, so that's all good. Thank you!

**Comment by reidpr** [comment]
> I hope you understand the intention of this style is to _improve_ usability, in the sense that it helps you identify which issues you have visited or not.

Absolutely! It didn't occur to me because the repos I interact with mostly are ones I'm in charge of (and, as you note, look at everything), but I totally get it once I read the background.

**Comment by kidonng** [comment]
This has been made into a standalone feature in the just released `21.1.18` version. It still applies globally (doesn't check if you have write access, sorry), but can be disabled with a single click.
